### ISSUE TITLE
How to use model "res_dna_r941_min_modbases-all-context_v001.cfg" from rerio

### ISSUE THREAD

**Issue by fsnibs10** [nanoporetech/remora#72]
Hi experts, 

I want to analyze my methylation sequencing data based on [Distinguishing between recruitment and spread of silent chromatin structures in Saccharomyces cerevisiae ](https://elifesciences.org/articles/75653). This paper mentioned that 

> modifications called with Megalodon (https://github.com/nanoporetech/megalodon, v2.3.3; Stoiber, 2021a) using the all-context rerio model (https://github.com/nanoporetech/rerio, res_dna_r941_min_modbases-all-context_v001.cfg; Stoiber, 2021b) and the flags --mod-motif ‘Y A 0’, --files_out ‘basecalls mod_mappings per_read_mods’

From the megalodon github,  it says there are new bioinformatic resources that can replace Megalodon. I would like to use Remora to generate modification result using the same model and parameters like paper.  So I have two questions:

1.  Is that feasible for Remora to use model from rerio? 
2.  How can I adjust paprameters as the same as the paper description?

Thanks!

**Comment by marcus1487** [comment]
The `res_dna_r941_min_modbases-all-context_v001.cfg` model is the older style "flip-flop modified base model" where the modified base calls and canonical basecalls come out of the same neural network. This setup is not optimal for a number of reasons. Thus we have moved to the Remora style models which are separated from the basecalling model allowing for more robust training sets while achieving the same accuracy. 

We do not intend to port the R9 series all-context model to the Remora model style as we are focused now on the kit14 chemistry and recommend that users upgrade to the new flowcells and kits.
